curriculum/challenges/english/01-responsive-web-design/basic-css-cafe-menu/part-071.md

```diff
@@ -1,22 +1,28 @@
 ---
-id: 5f459a7ceb8b5c446656d88b
+id: 5f459cf202c2a3472fae6a9f
 title: Part 71
 challengeType: 0
 dashedName: part-71
 ---
 
 # --description--
 
-Notice the grey color along the edges of the line. Those edges are known as <dfn>borders</dfn>. Each side of an element can have a different color or they can all be the same.
+Notice how the thickness of the line looks bigger? The default value of a property named `border-width` is `1px` for all edges of `hr` elements. By changing the background to the same color as the background, the total height of the line is `5px` (`3px` plus the top and bottom border width of `1px`).
 
-Make all the edges of the `hr` element the same color as the background of it using the `border-color` property.
+Change the `height` property of the `hr` to be `2px`, so the total height of it becomes `4px`.
 
 # --hints--
 
-Test 1
+You should set the `height` property to `2px`.
 
 ```js
+assert(code.match(/height:\s*2px;?/i));
+```
+
+Your `hr` element should have a `height` of `2px`.
 
+```js
+assert($('hr').css('height') === '2px');
```

Comment:
this does not pass with the seed from the next challenge:
```css
hr {
  height: 2px;
  background-color: brown;
  border-color: brown;
}

```

Reply:
Hmm, the height assertions break on zoom levels other than 100% - I see the issue now.